Eclecticiq-ic-app/default/data/ui/views/setup_view_dashboard.xml

```diff
@@ -0,0 +1,19 @@
+<!-- 
```

Comment:
This file is not not required. Remove it.

Reply:
Okay